README.md

```diff
@@ -23,6 +23,14 @@ This sets up a new WSL distribution `NixOS` that is installed under
 `.\NixOS`. `nixos-wsl-installer.tar.gz` is the path to the file you
 downloaded earlier. You might need to change this path or change to the download directory first.
 
+<details>
+    <summary>Did Install hang with "Starting Systemd.." or do you get error message `nsenter: failed to parse pid: '-p'`?</summary>
```

Comment:
Using markdown code blocks does not work inside of html code

Reply:
Thanks for feedback @SuperSandro2000 , should be better now I hope.